src/GraphQL/GraphQLBuilderExtensions.cs

```diff
@@ -219,6 +219,42 @@ public static IGraphQLBuilder AddSchema<TSchema>(this IGraphQLBuilder builder, F
         }
         #endregion
 
+        #region - AddAutoSchema / WithMutation / WithSubscription -
+        public static IGraphQLBuilderOrConfigureAutoSchema AddAutoSchema<TQueryClrType>(this IGraphQLBuilder builder)
+        {
+            builder.AddSchema<Schema>(ServiceLifetime.Singleton);
+            builder.Services.TryRegister<IGraphTypeMapping, AutoRegisteringGraphTypeMapping>(ServiceLifetime.Singleton);
+            builder.ConfigureSchema((schema, provider) => schema.Query = provider.GetRequiredService<AutoRegisteringObjectGraphType<TQueryClrType>>());
+            return new GraphQLBuilderOrConfigureAutoSchema(builder);
+        }
+
+        public static IGraphQLBuilderOrConfigureAutoSchema WithMutation<TMutationClrType>(this IGraphQLBuilderOrConfigureAutoSchema builder)
+        {
+            builder.ConfigureSchema((schema, provider) => schema.Query = provider.GetRequiredService<AutoRegisteringObjectGraphType<TMutationClrType>>());
```

Comment:
```suggestion
            builder.ConfigureSchema((schema, provider) => schema.Mutation = provider.GetRequiredService<AutoRegisteringObjectGraphType<TMutationClrType>>());
```
oops, for subscription as well

Reply:
done